### PR TITLE
Documents the Security Solution tag

### LINKED ISSUE #3366: Labeling of Dashboards of Kibana into Security Solution is not documented
## Description 

In the existing documentation I nowhere see even mentioned that tag `Security Solution` is responsible to move a given dashboard into security solution. This is already existing since 8.4, but until today I don't see that documented anywhere...
I would expect that at least that at least on these particular places:
https://www.elastic.co/guide/en/kibana/8.8/create-a-dashboard-of-panels-with-web-server-data.html#_save_the_dashboard
https://www.elastic.co/guide/en/kibana/8.8/dashboard.html#add-dashboard-settings
would be a link to the security overview page:
https://www.elastic.co/guide/en/security/8.8/overview-dashboard.html

On the latter page is then completely missing the Custom view that holds the dashboards. And there in that new section we could describe to use the `Security Solution` tag in order to move a dashboard to there.
And then we should create as well a subpage that describes how to deal with custom dashboards in Security solution and show some screenshots how to view data into Timeline since 8.8.

From my perspective - even if that particular request now seems more like a Elastic wide effort - it would be cool if we mentioned on this page what every single built in kibana tag is for:
https://www.elastic.co/guide/en/kibana/8.8/managing-tags.html
I did an inventory of my tags, but I am not sure if there are not ones that I created so that's to be checked...
```
  experimental
  Host Risk Score - default
  Infoblox NIOS
  Kubernetes
  Managed
  Osquery Manager
  Security Solution
  System
  Threat Intelligence Utilities
  Threat Intelligence
  User Risk Score - default
```

## Acceptance Test Criteria

At the end it should be transparent how to get a view to the Kibana dashboard view with the new Portable Dashboards feature in Security UI.

cc ^^ @paulewing

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
- Resolves #3366 

Preview: [Dashboards](https://security-docs_3739.docs-preview.app.elstc.co/guide/en/security/8.8/dashboards-overview.html)

_Note_: This is already documented in [8.9](https://www.elastic.co/guide/en/security/current/dashboards-overview.html). This PR backports the change to versions 8.4—8.8.

### CODE REVIEW

File: docs/dashboards/dashboards-overview.asciidoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 The following sections describe the {security-app}'s prebuilt dashboards, which provide visualizations of your security environment.
 
-You can also create and access custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard**. Once created, custom dashboards appear on the page:
+You can also create and access custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard**. Custom dashboards must have the tag `Security Solution` to appear on the Dashboards page.
```
Comment (benironside) [inline docs/dashboards/dashboards-overview.asciidoc:7]:
```suggestion
You can also create and access custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard**. For a custom dashboard to appear on the Dashboards page, it must have the tag `Security Solution`.
```

**Review by benironside — COMMENTED** [review]
Left one idea for your consideration. Feel free to incorporate or resolve my suggestion.

**Review by nastasha-solomon — APPROVED** [review]
Small nit for

**Comment by natasha-moore-elastic** [comment]
@Mergifyio backport 8.4 8.5 8.6 8.7 8.8

**Comment by natasha-moore-elastic** [comment]
@Mergifyio backport 8.4 8.5 8.6 8.7 8.8

**Pull request merged**